packages/esm-patient-banner-app/src/contact-details/contact-details.component.tsx

```diff
@@ -131,12 +131,16 @@ const Contact: React.FC<{ telecom: Array<fhir.ContactPoint>; patientUuid: string
         <InlineLoading description={`${t('loading', 'Loading')} ...`} role="progressbar" />
       ) : (
         <ul>
-          {contactAttributes?.length > 0 ? (
-            contactAttributes.map(({ attributeType, value, uuid }) => (
-              <li key={uuid}>
-                {attributeType.display} : {value}
-              </li>
-            ))
+          {value ? (
+            <React.Fragment>
+              <li>{value}</li>
```

Comment:
Restoring `value` here is the actual fix. It must've been removed at some point. I've added a test to assert that this is present, which should hopefully catch a regression in future.